src/osd/OSDMap.h

```diff
@@ -275,12 +275,10 @@ class OSDMap {
   }
 
   // no copying
-  /* oh, how i long for c++11...
 private:
   OSDMap(const OSDMap& other) = default;
```

Comment:
Yes Kefu, I believe the intent was that they would only be usable within the OSDMap class and unusable from outside of that class but I'll let @liewegas and/or @athanatos comment on that. Happy to make the change if required of course and thanks for taking a look.